_topic_map.yml

```
@@ -3046,16 +3048,20 @@ Topics:
     File: serverless-ossm-custom-domains
   - Name: Configuring TLS for a custom domain using Service Mesh
     File: serverless-ossm-tls
+#
 # Integrations
 - Name: Integrations
   Dir: integrations
   Topics:
   - Name: Using NVIDIA GPU resources with serverless applications
     File: gpu-resources
-# CLI reference guide
-- Name: CLI reference
-  Dir: cli_reference
+#
+# CLI tools
+- Name: CLI tools
+  Dir: cli_tools
   Topics:
+  - Name: Installing the Knative CLI
+    File: installing-kn
   - Name: Advanced kn configuration
```

I'd vote for dropping "kn" from the title, but it's purely my subjective perspective.
```suggestion
  - Name: Advanced configuration
```

---

Probably I do like a bit more to see `the Knative CLI` in the titles, yet again it's just a suggestion to consider in a wider group. :)

---

I have fixed these in the other PR for domain mapping 🙂 so this can be rebased on that